Composer/packages/form-dialogs/src/FormDialogSchemaEditor.tsx

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+// eslint-disable-next-line @typescript-eslint/camelcase
+import { RecoilRoot, useRecoilTransactionObserver_UNSTABLE } from 'recoil';
+import { formDialogSchemaJsonSelector } from 'src/atoms/appState';
+import { useHandlers } from 'src/atoms/handlers';
+import { VisualEditor } from 'src/components/VisualEditor';
+
+export type FormDialogSchemaEditorProps = {
+  /**
+   * Unique id for the visual editor.
+   */
+  editorId: string;
+  /**
+   * Wether to show or hide the theme picker.
+   */
+  showThemePicker?: boolean;
```

Comment:
What is a theme picker for this UI?

Reply:
Good catch, I removed theming infra and forgot to remove this